apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx

```diff
@@ -206,9 +199,9 @@ This assertion contains information about the user's identity and other authenti
 - Department or organization
 - Other attributes present in the users directory managed by the identity provider
 
-Other than the unique ID of the user, SAML does not make it mandatory that any other attributes appear in the assertion. Identity Providers are configured about what user information is shared with your project.
+Other than the unique ID of the user, SAML does not make it mandatory that any other attributes appear in the assertion. Identity providers are configured about what user information is shared with your project.
```

Comment:
/nit 
```suggestion
With exception of the unique user ID, SAML does not require any other attributes in the assertion. Identity providers can be configured so that only select user information is shared with your project.
```